pkg/hooks/monitor.go

```diff
@@ -2,9 +2,8 @@ package hooks
 
 import (
 	"context"
-	"os"
-	"path/filepath"
 
+	current "github.com/containers/libpod/pkg/hooks/1.0.0"
```

Comment:
This seems like a weird name, why not have it.

hooks "github.com/containers/libpod/pkg/hooks/1.0.0"

Reply:
@rhatdan, agreed, but I used it to stay consistent with other usages in other files, where it is also named 'current'. Think I should change it here and everywhere else?

Reply:
I suggest to stick with `current` in this PR. We can clean this up later and if desired open an issue to not forget about it.